meson.build

```diff
@@ -392,12 +394,14 @@ features += {'cocoa': cocoa.found()}
 if features['cocoa']
     dependencies += cocoa
     sources += files('osdep/language-mac.c',
-                     'osdep/main-fn-mac.c',
                      'osdep/path-mac.m',
                      'osdep/utils-mac.c',
                      'osdep/mac/application.m',
                      'osdep/mac/events.m',
                      'osdep/mac/menubar.m')
+    if get_option('cplayer')
```

Comment:
cplayer check here is pointless

Reply:
The variable is only used in a section guarded by `if get_option('cplayer')` at the end and there's no harm in setting something that ends up unused,